### DISCUSSION TITLE
does kratos have a admin web?

### DISCUSSION THREAD

**Discussion by youkechat** [ory/kratos#2584] [Q&A] [answered]
Does kratos have a admin web? if don't have, request support it, with feature such as kick user out, delete user and so on

**Comment by vinckr** [comment, marked as answer]
Hello @youkechat 

Ory Kratos is completely headless. You can manage everything you mentioned through the API and CLI directly. 
If you are looking for a graphical user interface to manage identities, I suggest to check out [Ory Cloud](https://console.ory.sh/registration) - this comes with an UI for all Ory services and has a free tier for developers. 
There are also some community projects to add simple interfaces to Kratos, check out this one: https://github.com/dfoxg/kratos-admin-ui

Let me know if that answers your question.
